while True:
    try:
        x = int(input("x: "))
        y = int(input("y: "))
        print(x/y)
    except(ZeroDivisionError, ValueError) as e:
        print("yanlış bilgi girdiniz")
        print(e)
    else:
        break
    finally:
        print("try except sonlandı")
        
#except Exception as ex:   de işe yarıyor ama hata türünü bilemiyoz

